### DISCUSSION TITLE
Docker Image

### DISCUSSION THREAD

**Discussion by Pompey69** [whyboris/Video-Hub-App#662] [General]
I'm trying to run this on my NAS

I was wondering if there is a docker image to do so?

Thanks!

**Comment by whyboris** [comment]
There's no Docker image currently or planned. It would take a lot of learning on my side to get it to work and I'd rather spend the time on features more people would benefit from.

If someone knows how to do it, thinks it's relatively simple, and is willing to help me, I'm happy to learn from you and get a Docker image working.

If your NAS can run a Docker image, it's likely a full Linux OS already -- you could just run the Linux build of the app 🤔 right?
